core/common/src/main/java/alluxio/underfs/UnderFileSystemRegistry.java

```diff
@@ -111,7 +111,8 @@ public static UnderFileSystem create(String path, Map<String, String> ufsConf) {
     for (UnderFileSystemFactory factory : factories) {
       try {
         // Use the factory to create the actual client for the Under File System
-        return new UnderFileSystemWithLogging(factory.create(path, ufsConf));
+        return new UnderFileSystemWithLogging(
+            factory.create(path, new UnderFileSystemConfiguration(ufsConf)));
```

Comment:
as talked offline, we agree the limitation of supporting only `Map<String, String>`, but leave this decision when we have better / real use case to guide in order to better support customizing ufs